src/App.tsx

```diff
@@ -4,32 +4,71 @@ import logo from "../public/FischLogo.svg";
 
 import styles from "./App.module.css";
 import { Card } from "./components/Card";
-import { useRef, useState } from "react";
+import { JSX, useEffect, useState } from "react";
+import { NewCard } from "./components/NewCard";
 
-function App() {
-  const [cardsAmount, setCardsAmount] = useState(1)
-  const [cards, setCards] = useState([{}]);
+interface AppProps {
+  basicCards: {
+    face: string;
+    back: string;
+  }[];
+}
+
+interface CardObject {
+  face: string;
+  back: string;
+}
+
+function App(props: AppProps) {
+  const importedCards = props.basicCards.map((card, index) => {
+    return { face: card.face, back: card.back, key: index };
+  });
+
+  const [cardsArray, setCardsArray] = useState(importedCards);
+
+  const [incrementalKey, setIncrementalKey] = useState(cardsArray.length);
```

Comment:
**nitpick:** You don't need it - let's remove it for the purpose of simplifying this code

Reply:
https://react.dev/learn/rendering-lists - this article you told me to read says I do. If I don't use key, everything is messed up: the cards are rendered in the wrong order, and I'm not able to distinguish which card is to be removed.

Reply:
You are right - but it's not how's it's done in React

Reply:
I had to read it a couple times to get it :)